src/pages/home/HomePage.tsx

```diff
@@ -29,7 +29,7 @@ export interface IResultsViewPageProps {
     routing: any;
 }
 
-export function createQueryStore(currentQuery?:any) {
+export function createQueryStore(currentQuery?:any, clear:boolean = true) {
```

Comment:
clearUrlOnSubmit?

Reply:
I think `clearQueryOnSubmit` is more appropriate. Do you agree?